modules/core/shared/src/main/scala/gem/math/MagnitudeValue.scala

```diff
@@ -10,7 +10,9 @@ import cats.instances.int._
  * Exact magnitude value represented as an int with the original value scaled up
  * @param value This magnitude integral value
  */
-final case class MagnitudeValue(val value: Int) extends Product with Serializable
+final case class MagnitudeValue(private[gem] val scaledValue: Int) extends Product with Serializable {
+  def toDouble: Double = scaledValue / 10.0
```

Comment:
Shouldn't this be `/ 100.0`?  When you read the value from the database it is scaled up by 10^2 so it should be scaled down here by the same.

Reply:
fixed